### ISSUE TITLE
Is there a way to get infinitive form of verbs? or tell the tenses of verbs?

### ISSUE THREAD

**Issue by charmingalgoman** [knuppe/SharpNL#20]
Hi.

Thank you for your great work.
I am a newer, so sorry for questioning some basic issues.
How to get helping documents for SharpNL?

Package Knuppe.SharpNL has been installed to project. Viewing ShaprNL namespace I found Inflector class which provides Pluralize() and Singularize().

Currently I need the function which can show what tense a verb is and what the infinitive form is. 
Is it possible to do that?

Thanks a lot

**Comment by knuppe** [comment]
Hi,

Unfortunately, there is not much arroud at this moment, I wrote a few pages on the [wiki](https://github.com/knuppe/SharpNL/wiki), but they do not cover the whole library. The [documentation](https://opennlp.apache.org/documentation.html) from the original java project is a good starting point.

It is possible to create a tool to detect the tense, but honestly I did not research about it yet... maybe in the future.

:+1:
